include/wx/osx/core/colour.h

```diff
@@ -29,60 +29,78 @@ class WXDLLIMPEXP_CORE wxColour: public wxColourBase
     // default copy ctor and dtor are ok
 
     // accessors
-    virtual bool IsOk() const { return m_cgColour != NULL; }
+    virtual bool IsOk() const wxOVERRIDE;
 
-    virtual WXDLLIMPEXP_INLINE_CORE ChannelType Red() const { return m_red; }
-    virtual WXDLLIMPEXP_INLINE_CORE ChannelType Green() const { return m_green; }
-    virtual WXDLLIMPEXP_INLINE_CORE ChannelType Blue() const { return m_blue; }
-    virtual WXDLLIMPEXP_INLINE_CORE ChannelType Alpha() const { return m_alpha; }
+    virtual ChannelType Red() const wxOVERRIDE;
+    virtual ChannelType Green() const wxOVERRIDE;
+    virtual ChannelType Blue() const wxOVERRIDE;
+    virtual ChannelType Alpha() const wxOVERRIDE;
+
+    wxColour& operator=(const wxColour& col);
```

Comment:
This shouldn't normally be necessary when using ref data. Why is it defined?

Reply:
right, no special assignment code needed anymore